evap/staff/templates/staff_semester_import.html

```diff
@@ -35,6 +39,9 @@ <h3>{% trans 'Import semester data' %}</h3>
     </form>
 {% endblock %}
 
- {% block additional_javascript %}
- {% include 'bootstrap_datetimepicker.html' %}
- {% endblock %}
+{% block additional_javascript %}
+    {% include 'bootstrap_datetimepicker.html' %}
+    <script type="text/javascript">
+        {% include "copy-to-clipboard.js"%}
```

Comment:
missing space before closing `%}` and please use `'` instead of `"` for the include